src/elements/header/spec/script.spec.js

```diff
@@ -0,0 +1,5 @@
+import header from '../script';
+
```

Comment:
The reason for that is mainly because of the .eslint config. I don't want the jest global on the other files.